libraries/botframework-streaming/src/namedPipe/namedPipeServer.ts

```diff
@@ -73,14 +74,15 @@ export class NamedPipeServer implements IStreamingTransportServer {
         }
 
         const incoming = new Promise(resolve => {
-            this._incomingServer = new Server((socket: Socket): void => {
-                this._receiver.connect(new NamedPipeTransport(socket));
-                resolve();
-            });
+                this._incomingServer =  createNodeServer((socket: INodeSocket2): void => {

```

Comment:
I'm very skeptical of code that doesn't have error conditions, especially around networking. But it doesn't look like this is due to your PR, so could be considered out of scope.

Reply:
So if you look at createNodeServer() it throws if an error occurs which the new Promise() will catch and automatically turn into a rejection. I would have made createNodeServer() return a promise instead but this code should work ok. 

---
In reply to: [384669694](https://github.com/microsoft/botbuilder-js/pull/1795#discussion_r384669694) [](ancestors = 384669694)